benchmark/dataset.py

```diff
@@ -41,9 +41,11 @@ def download(self):
             print(f"Downloading {self.config.link}...")
             tmp_path, _ = urllib.request.urlretrieve(self.config.link)
 
-            if tmp_path.endswith(".tgz") or tmp_path.endswith(".tar.gz"):
+            if self.config.link.endswith(".tgz") or self.config.link.endswith(
+                ".tar.gz"
+            ):
                 print(f"Extracting: {tmp_path} -> {target_path}")
-                (DATASETS_DIR / self.config.path).mkdir(exist_ok=True)
+                (DATASETS_DIR / self.config.path).mkdir(exist_ok=True, parents=True)
```

Comment:
It was intentional not to include parents

Reply:
Does that make any issues during further processing?